.gitignore

```diff
@@ -18,3 +18,4 @@ coverage.txt
 test/e2e/coverage.txt
 apisix-ingress-controller
 .actions/openwhisk-utilities
+conf/config.yaml
```

Comment:
Please check the editor, the EOL symbol is required.

Reply:
solved, i remove the change of .gitignore